benchmarks/views/index.py

```diff
@@ -109,7 +109,7 @@ def _collect_benchmarks():
                 instance.children = [child.value.identifier for child in node.children]
                 traverse_todo = node.children + traverse_todo
             else:  # no children --> it's a specific instance
-                instance = BenchmarkInstance.objects.select_related('benchmark_type__reference') \
+                instance = BenchmarkInstance.objects.select_related('benchmark_type') \
```

Comment:
the idea was to pre-fetch the benchmark_type's reference as well with `benchmark_type__reference` for efficiency, is that not necessary?